manifests/init.pp

```diff
@@ -423,6 +449,13 @@
     fail('Values for dm_ variables will be ignored unless storage_driver is set to devicemapper.')
   }
 
+  if($tls_enable) {
```

Comment:
It would also be nice to warn if `tls_*` are set but `tls_enable` is set to false, as they then get silently ignored.